core/src/main/scripts/importer/cbioportalImporter.py

```diff
@@ -394,12 +394,7 @@ def check_dir(study_directory):
         print('Study cannot be found: ' + study_directory, file=ERROR_FILE)
         sys.exit(2)
 
-def interface():
-    parser = argparse.ArgumentParser(description='cBioPortal meta Importer')
-    parser.add_argument('-c', '--command', type=str, required=False,
```

Comment:
It would be nice to offer backwards compatibility for the `-c` parameter. I.e. if someone supplies that, call the relevant command accordingly. Maybe add in the help text that the recommended way is to use the command directly and that this is only to provide backwards compatibility